types/src/transaction/transaction.rs

```diff
@@ -264,6 +232,115 @@ fn is_input_and_output_consistent(inputs: &[AssetTransferInput], outputs: &[Asse
     sum.iter().all(|(_, sum)| sum.is_zero())
 }
 
+fn make_tag(mut item: Vec<u8>, mut result: u128) -> u128 {
+    if item.is_empty() {
+        return result
+    }
+    // Assume that the mask is stored as little endian
```

Comment:
Usually, a bitmask uses a big-endian. Why do you use a little-endian?

Reply:
I just specified the way how u128, the bitmask, is stored in Vec<u8>. I did it cause it was a more familiar way. I change it to stored as big-endian.